### PR TITLE
Gt gem gone

### PR DESCRIPTION
**Pull request by gary1410**
@roberttaraya

### CODE REVIEW

**Comment by gary1410** [comment]
@roberttaraya meeeerge

File: app/assets/javascripts/scripts.js [hunk 1]
```diff
@@ -8,26 +8,34 @@ function initializeMap() {
...
-  var wineryLocations = locations // locations was defined when winery#index was called
+  mapLoop( map, bounds );
 
-  for ( i = 0 ; i < wineryLocations.length ; i++ ) {
-    var title = wineryLocations[i][0];
-    var position = new google.maps.LatLng(wineryLocations[i][1], wineryLocations[i][2]);
-    bounds.extend(position);
+  }
+
+  function mapLoop (map, bounds) {
```
Comment (roberttaraya) [inline app/assets/javascripts/scripts.js:22]: what the hell kind of name is mapLoop LOL

**Comment by roberttaraya** [comment]
ok...looks fine to me. merge away

**Comment by roberttaraya** [comment]
nevermind...i'll do it this time

**Pull request merged**